src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -175,6 +175,16 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_IncludeRazorLanguageConfiguration" DependsOnTargets="PrepareForBuild" BeforeTargets="CoreCompile">
+    <ItemGroup>
+      <Content Include="..\Microsoft.AspNetCore.Razor.VSCode.Extension\language-configuration.json">
```

Comment:
Since this is now shared between VS and VSCode might it not be more appropriate to put it in a root or shared directory instead of leaving it in VSCode, and have them both copy it down? That helps communicate to anyone modifying it that they might have to check that it didn't break anything in multiple places (even if they should both be following the spec).